src/routes/index.js

```diff
@@ -7,18 +7,21 @@ import CreateArticle from "../components/articles/createArticle";
 import NewPassword from "../components/auth/passwordReset/newPassword";
 import ResetRequest from "../components/auth/passwordReset/passwordResetPage";
 import ProtectedRoute from "./protectedRoutes";
+import Rating from "../components/articles/rating";
 
+const expressHistory = "pushState" in window.history;
 export default function AppRoutes() {
   return (
     <div>
-      <BrowserRouter>
+      <BrowserRouter forceRefresh={expressHistory}>
```

Comment:
After merging the view an article with this task, try to do away with the forceRefresh attribute

Reply:
@LehruAngela I have a function that should run when a user clicks on an article. In that function, I check if that user has ever rated that article or if that I user is the author of that article. The only way that function works is if I add the force refresh. I tried looking for other ways the whole of yesterday but I failed